atat_internal_api.yaml

```diff
@@ -0,0 +1,2197 @@
+openapi: 3.0.2
+info:
+  description: >-
+    This is the ATAT Internal API. It has operations for creating, updating and destroying ATAT resources, as well as
+    for obtaining reporting data on provisioned Portfolios.
+  version: 1.0.0
+  title: ATAT Internal API
+  contact:
+    email: atat-dev+provisioning_api@ccpo.mil
+tags:
+  - name: CRUD
+    description: >-
+      Operations related to managing Portfolios, Applications and Environments
+paths:
+  /portfolios:
+    get:
+      operationId: getPortfolios
+      parameters:
+        - $ref: '#/components/parameters/offsetParam'
```

Comment:
Is there any way to add a parameter that will support returning  portofolios based on lifecycle_status (Active, Draft, Archived)?

Reply:
We'll be designing the filtering and sorting capabilities soon. Most likely, it'll either be a `q` parameter which will contain a full query object, or specifically named set of queries for each filter that we need to support.